packages/ui/src/components/typeahead/Typeahead.tsx

```diff
@@ -112,8 +112,9 @@ export const Typeahead: FunctionComponent<TypeaheadProps> = ({
   const filteredItems = useMemo(
     () =>
       items.filter((item) => {
-        const hasNameMatch = item.name?.includes(inputValue);
-        const hasDescriptionMatch = item.description?.includes(inputValue);
+        const lowerFilterValue = inputValue.toLowerCase();
```

Comment:
Better to move this variable outside of the scope, so we don't need to lowercase it on every element.